Apps/Contoso.Android.Puppet/MainActivity.cs

```diff
@@ -49,12 +52,23 @@ protected override void OnCreate(Bundle savedInstanceState)
             Crashes.ShouldAwaitUserConfirmation = ConfirmationHandler;
 
             Distribute.ReleaseAvailable = OnReleaseAvailable;
-
             AppCenterLog.Assert(LogTag, "AppCenter.Configured=" + AppCenter.Configured);
             AppCenter.SetLogUrl("https://in-integration.dev.avalanch.es");
             Distribute.SetInstallUrl("http://install.asgard-int.trafficmanager.net");
             Distribute.SetApiUrl("https://asgard-int.trafficmanager.net/api/v0.1");
-            AppCenter.Start("bff0949b-7970-439d-9745-92cdc59b10fe", typeof(Analytics), typeof(Crashes), typeof(Distribute));
+
+            // Enable Firebase Analytics if set
+            var enableAnalytics = Preferences.SharedPreferences.GetBoolean(Constants.FirebaseAnalyticsEnabledKey, false);
+            if (enableAnalytics)
+            {
+                Push.EnableFirebaseAnalytics();
+            }
+
+            Push.PushNotificationReceived -= PrintNotification;
```

Comment:
Is this needed even when the method is static?
Should we investigate a bit more? Customers will hit this problem too and we don't document that.

Reply:
I only did that because the explanation for doing it in a static constructor is to avoid duplicate handlers, and i didn't use a static constructor

Reply:
Ok, that's not what I meant.

If we really have an (ux) issue in our test app with that handler, there is a good chance customers will also have the issue, and it's either worth documenting or investigating to see if we can improve the consumption of this API.